packages/orbit-components/src/Alert/index.tsx

```diff
@@ -134,7 +134,7 @@ const Alert = (props: Props) => {
         "rounded-150 text-ink-dark font-base text-normal p-300 relative box-border flex w-full border border-t-[3px] leading-normal",
         "lm:border-s-[3px] lm:border-t",
         "tb:rounded-100",
-        inlineActions && "items-center",
+        inlineActions ? "items-center" : "",
```

Comment:
I added an empty string because before that, the default value in the background was `0n`. After upgrade, I got this error:

```
Argument of type 'false | "" | 0 | 0n | "justify-between" | null | undefined' is not assignable to parameter of type 'ClassValue'. Type '0n' is not assignable to type 'ClassValue'.ts(234)
```

Reply:
It would be preferred to do `Boolean(inlineActions) && "items-center"`